app/generator/content/delinked-help-info.js

```diff
@@ -11,7 +11,7 @@ const delinkedHelpInfo = (_delinkedStatement) => {
           { text: 'Your next payment ', style: 'header2' },
           '\n',
           { text: 'You can find out when you\'ll receive your next payment and information on progressive reductions for 2025 to 2027 (when they\'re available) at ' },
-          { text: 'www.gov.uk/guidance/delinked-payments-replacing-the-basic-payment-scheme', link: 'https://www.gov.uk/guidance/delinked-payments-replacing-the-basic-payment-scheme', decoration: 'underline', unbreakable: true }
+          { text: 'www.gov.uk/guidance/delinked-payments-replacing-the-basic-payment-scheme.', link: 'https://www.gov.uk/guidance/delinked-payments-replacing-the-basic-payment-scheme', decoration: 'underline', unbreakable: true }

```

Comment:
No problem with the full stop but I would say in general the full stop in a link is not usually also a link. Any way we can code around that or is it not feasible?

Reply:
discussed